app/controllers/assignments_controller.rb

```diff
@@ -33,12 +36,25 @@ def index
   end
 
   def new
-    @assignment = Assignment.new
-    @assignment.course_id = @course.id
-    @assignment.due_date = (Time.now + 1.week).end_of_day.strftime("%Y/%m/%d %H:%M")
+    @files = Assignment.new
```

Comment:
Exam doesn't need lateness_config, nor does a default lateness of "in a week" make sense. In this PR, Files has request_time_taken set by default.

This goes with splitting the _form into three forms, one for each type. Goal was to have clean form data for Rails, but the resulting UI is a bit cleaner too.